internal/providers/testing/provider_mock.go

```diff
@@ -379,7 +379,7 @@ func (p *MockProvider) PlanResourceChange(r providers.PlanResourceChangeRequest)
 		// computed+optional attributes has become unset
 		configVal, err := path.Apply(r.Config)
 		if err != nil {
-			return v, err
+			return v, nil
```

Comment:
What's the reason for this change?

It looks like this is the only place the callback could return an error which is checked for on line 399.

Reply:
If there's an error, it's just a value we can't do anything with, so the returning the same value is just as good. This is only a helper to create a default plan value that is valid, any test that needs a specific plan value will create it more directly. I only put that in the commit message though, I should add a comment ;)